### PR TITLE
Package tofn.0.1

### PR DESCRIPTION
**Pull request by matt-kukla**
### `tofn.0.1`
Typed ordered fuzzy numbers

---
* Homepage: https://mkukla.net/doc/tofn/
* Bug tracker: Matthew Kukla <matt.kukla@yandex.ru>

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/tofn/tofn.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+license: "GPL-3.0"
+homepage: "https://mkukla.net/doc/tofn/"
+dev-repo: "git+https://github.com/matt-kukla/tofn.git"
+depends: [
+  "ocaml"
+  "dune" {>= "2.9"}
+]
+build: [["dune" "build" "-p" name "-j" jobs "@install" ]]
+url {
+  src: "https://github.com/matt-kukla/tofn/archive/0.11.tar.gz"
```
Comment (kit-ty-kate) [inline packages/tofn/tofn.0.1/opam:15]: Is this expected? Should this package be version `0.11` instead?

Comment (kit-ty-kate) [reply packages/tofn/tofn.0.1/opam:15]: Also the `dune-project file and opam files got moved to the `src` directory in this tag so installing will not do anything.

Comment (matt-kukla) [reply packages/tofn/tofn.0.1/opam:15]: Yes, you're correct, it should be 0.11 instead, I had forgotten to update the version number.  Thanks.

Comment (matt-kukla) [reply packages/tofn/tofn.0.1/opam:15]: > Also the `dune-project` file and opam files got moved to the `src` directory in this tag so installing will not do anything.

Also correct, I've screwed up a number of things with this release.  Would it be easier if I opened a new pull request with the updated (and properly placed) dune/opam files, as version 0.12?

Comment (kit-ty-kate) [reply packages/tofn/tofn.0.1/opam:15]: yes, that'd be good. Could you also return https://github.com/ocaml/opam-repository/pull/19624/commits/ae7957f6915446607838cfd747c75a160722e269 upstream?

**Pull request closed without merging**